src/Grpc.AspNetCore.Server/Internal/GrpcEventSource.cs

```diff
@@ -23,7 +23,7 @@
 
 namespace Grpc.AspNetCore.Server.Internal;
 
-internal class GrpcEventSource : EventSource
+internal sealed class GrpcEventSource : EventSource
```

Comment:
👍 , although with the `static readonly Log` I would *hope* that the JIT devirtualizes anyway

Reply:
> 👍 , although with the `static readonly Log` I would _hope_ that the JIT devirtualizes anyway

Yes, jit does that for static readonly